lib/saint.js

```diff
@@ -79,15 +79,23 @@ p.getJobSegment = function(fileId, pageNum, callback){
 	});
 }
 
-p.importJob = function(jobDescription, jobData, callback){
-	var self = this,
-			populateData =[];
-	while(jobData.length){
-		populateData.push(jobData.splice(0,10000));
+p.importJob = function(jobDescription, jobData, callback, jobId){
+	var self = this;
+	var populateData = spliceJobData(jobData);
+	function spliceJobData(jobData) {
+		var splicedJobData = [];
+		while(jobData.length){
+			splicedJobData.push(jobData.splice(0,10000));
```

Comment:
So, if job data has an array longer then 10k we'd loose some data?

Reply:
When I read this I thought splicedJobData would end up as multi dimensional array of job data where each splice has less than 10000 rows.  There is another bit of code in classify-main on the tracking first side which splits jobs up into smaller pieces, 2000 rows I think, so this is likely never used. My only change here was moving the code into a function. I'm not sure if there's much benefit in spending much more time on this given it's a legacy app which will likely be retired this year.